arquillian-ape-nosql/couchbase/src/main/java/org/arquillian/ape/nosql/couchbase/CouchbasePopulatorService.java

```diff
@@ -15,7 +15,7 @@
 
 import static org.awaitility.Awaitility.await;
 
-class CouchbasePopulatorService implements NoSqlPopulatorService<Couchbase> {
+public class CouchbasePopulatorService implements NoSqlPopulatorService<Couchbase> {
```

Comment:
Maybe if we follow SPI approach proposed in the comment in the rule itself we won't have to make those classes public?